### PR TITLE
Add capitalization information

### LINKED ISSUE #908: Style Guide Update
In our handbook, we are using a mix of title case and sentence case capitalization. What is the preferred way and can we update the style guide?

### PR DESCRIPTION
**Pull request by sharjeelaziz**
The purpose of this PR is to invite discussion and agree on a capitalization style for the handbook. This has been discussed in Slack which leaned towards using sentence case. Closes #908

[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/sharjeelaziz-patch-1/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/000-contributing/writing-style-guide.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 
+## How to capitalize titles and headings
+
+We use sentence case for our titles and headings. When using sentence case, we capitalize the first word, all proper nouns, and a word following a colon. We do not capitalize the next word if a title or heading begins with a number.
+
+Examples
+
+- Site building using Drupal
+- Engineer's role in client relationships
+- 30 days checkin
```
Comment (mgifford) [inline docs/000-contributing/writing-style-guide.md]: I think it is either "check in" or "check-in" but not "checkin"

File: docs/000-contributing/writing-style-guide.md [hunk 2]
```diff
@@ -4,6 +4,16 @@
...
 This will be fleshed out in future iterations. For now, keep in mind that content appearing in the handbook should be welcoming, personable, and free from jargon.
 
+## How to capitalize titles and headings
+
+We use sentence case for our titles and headings. When using sentence case, we capitalize the first word, all proper nouns, and a word following a colon. We do not capitalize the next word if a title or heading begins with a number.
+
+Examples
+
+- Site building using Drupal
+- Engineer's role in client relationships
```
Comment (mgifford) [inline docs/000-contributing/writing-style-guide.md:14]: Maybe drop this example and replace it with one with a colon. 
"- Accessibility: Everyone has a role"

**Comment by mgifford** [comment]
I made a couple short comments, but just wanted to check to see if there are any style guides we can cite as part of this. 

Closest government example of this I could find was https://content-guide.18f.gov/our-style/capitalization/

File: docs/000-contributing/writing-style-guide.md (hunk 1 above)
Comment (sharjeelaziz) [reply docs/000-contributing/writing-style-guide.md]: Thanks for catching that.

File: docs/000-contributing/writing-style-guide.md (hunk 2 above)
Comment (sharjeelaziz) [reply docs/000-contributing/writing-style-guide.md:14]: Nice. Done.

**Comment by ContentBrewmaster** [comment]
Both Suzanne Gray and I agree that sentence case is preferable as a style for the handbook.

**Comment by mgifford** [comment]
And the whole instructions provided make sense to you too @ContentBrewmaster ?

**Comment by ContentBrewmaster** [comment]
@mgifford yes the full instructions makes sense. I think this can be deployed @dmundra

**Comment by dmundra** [comment]
Thanks @ContentBrewmaster just need you or @mgifford to approve this pull request.

**Review by ContentBrewmaster — APPROVED** [review]

**Pull request merged**